lib/state/init.js

```diff
@@ -1,19 +1,30 @@
 'use strict';
 const _ = require('lodash');
 const path = require('path');
-const globalModulesPath = require('global-modules');
 const readPackageJson = require('../in/read-package-json');
 const globalPackages = require('../in/get-installed-packages');
 const emoji = require('../out/emoji');
+var globalModulesPath = require('global-modules');
```

Comment:
Can you convert this back to a `const`, and preferably move it back up to the same position so that it's not showing up in the diff...